articles/keywords-and-identifiers.html

```diff
@@ -34,16 +34,17 @@ <h3>Keywords</h3>
 <li>
 	<code>break</code>, <code>case</code>, <code>continue</code>, <code>default</code>, <code>else</code>, <code>fallthrough</code>,
 	<code>for</code>, <code>goto</code>, <code>if</code>, <code>range</code>, <code>return</code>, <code>select</code> and <code>switch</code>
-	are used to control code flows.
+	are used to control flow keywords.
 </li>
 <li>
-	<code>defer</code> and <code>go</code> can also be viewed as control flow keywords.
-	But they are some special. They are modifiers of function calls.
+	<code>defer</code> and <code>go</code> are also control flow keywords, but in a specific manner.

```

Comment:
"in other specific manners" is better I think, as "defer" and "go" are two different control flow mechanisms.